File: 2022.01.19/javascript/1929.js
const [m,n] = require('fs').readFileSync('/dev/stdin').toString().split('\n')[0].split(' ').map(Number);
let prime = new Array(n+1).fill(true);
let j;
let result = "";
prime[0] = prime[1] = false;
for (let i = 2 ; i<parseInt(Math.sqrt(n))+1 ; i++){
    j = 2;
    while (i*j<=n){
        prime[i*j] = false;
        j++;
    }
}
for (let i = m ; i<=n ; i++){
    if (prime[i]) result += i+"\n"
}
console.log(result)

File: 2022.01.19/python/1929.py
m,n = map(int,input().split())
prime = [True for _ in range(n+1)]
prime[0] = prime[1] = False
for i in range(2,int(n**0.5)+1):
    j = 2
    while i*j<=n:
        prime[i*j]=False
        j+=1
for i in range(m,n+1):
    if prime[i]: print(i)